prost-types/src/lib.rs

```diff
@@ -9,9 +9,17 @@
 //!
 //! [1]: https://developers.google.com/protocol-buffers/docs/reference/google.protobuf
 
-use std::convert::TryFrom;
-use std::i32;
-use std::i64;
+#![no_std]
+extern crate alloc;
+#[cfg(feature = "std")]
+extern crate std;
```

Comment:
are these `extern crate` lines necessary?  I haven't used no-std extensively, but it seems like they shouldn't be

Reply:
I think they are, we did the exact same thing in bytes crate (your dependency):

https://github.com/tokio-rs/bytes/blob/master/src/lib.rs#L76

Reply:
here was the discussion in that PR: https://github.com/tokio-rs/bytes/pull/281#discussion_r320497569

Reply:
`extern crate alloc` and `extern crate std` are the last two things that cannot be done any other way besides an `extern crate` directive